### PR TITLE
Repository structure

### PR DESCRIPTION
**Pull request by rubenVB01**

### CODE REVIEW

**Review by lantoli — APPROVED** [review]
shouldn't we fill some files like READM.md?

also not sure about exampleA/B as it looks like it will be removed/renamed.

similar for .tf files, i feel like it would be better to include them in the repo the first time we need them

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+Please include a summary of the fix/feature/change, including any relevant motivation and context.
+
+Link to any related issue(s):
+
+## Type of change:
+
+- [ ] Bug fix (non-breaking change which fixes an issue). Please, add the "bug" label to the PR.
+- [ ] New feature (non-breaking change which adds functionality). Please, add the "enhancement" label to the PR. A migration guide must be created or updated if the new feature will go in a major version.
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected). Please, add the "breaking change" label to the PR. A migration guide must be created or updated.
+- [ ] This change requires a documentation update
```
Comment (EspenAlbert) [inline .github/pull_request_template.md:12]: Is this necessary?

**Comment by EspenAlbert** [comment]
agree with @lantoli here. I would create a minimal terraform module instead of empty files

File: .github/pull_request_template.md (hunk 1 above)
Comment (rubenVB01) [reply .github/pull_request_template.md:12]: @lantoli @EspenAlbert At first this ticket just has the goal to set up the repository structure (about the examples, it is true that exampleA/exampleB will be deleted in the future, was also just to show that each example is represented as a subdirectory, but I can just leave the "examples" directory). 

The implementation of the module (i.e., the implementation of main, variables, outputs, versions .tf files) and the redaction of the license and the readme are in separated tickets. If it does not make sense to do this small PR just with the structure (empty) I can add some implementation/documentation here already.

Comment (rubenVB01) [reply .github/pull_request_template.md:12]: @EspenAlbert About the pull_request_template, I took the same from the provider. If that is not necessary at all I can remove it.

Comment (EspenAlbert) [reply .github/pull_request_template.md:12]: Up to you. I think it is a bit noisy.
I would at least remove: `- [ ] This change requires a documentation update`, but just my opinion 😅

**Pull request merged**